net-libs/gloox/gloox-1.0.28-r1.ebuild

```
@@ -50,4 +50,6 @@ src_configure() {
 src_install() {
 	default
 	find "${ED}" -name "*.la" -delete || die
+
+	use examples && dodoc -r "${WORKDIR}"/${P}/src/examples/
```

```suggestion
	use examples && dodoc -r "${S}"/src/examples/
```

or even

```suggestion
	use examples && dodoc -r src/examples/
```
should work.